### PR TITLE
To Do List-List Structure

### PR DESCRIPTION
**Pull request by harshi0102**
In the first phase of the project the following steps have been completed:

Setup webpack
Create index.html with empty To-Do List placeholders elements
Create an index.js file and set an array of some simple to-do tasks (array of objects). Each task object contains three keys:
-- description [string].
-- completed [boolean].
-- index: [number].
Write a function to iterate over the tasks array and populate an HTML element for each task.
On page load render the dynamically created tasks in the dedicated placeholder. The list appears in order of the index values for each task.
Create a style.css and set rules for the To-Do List. CSS must be loaded by Webpack Style/CSS Loader. The list should be a clone of the part of the minimalist project captured in the video shown in the README.md file.
All source files (index.html, index.js, and style.css) are located in /src directory and distribution files are generated by webpack and served by the webpack dev server from /dist folder.

### CODE REVIEW

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @harshi0102 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 

You have submitted a project, that lacks the structure of the README given. Kindly add the section headers, and every other section you omitted such as features and future features. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  taskList.innerHTML = '';
+
+  tasks.forEach((task, index) => {
+    const taskElement = document.createElement('li');
+    taskElement.className = 'task';
+    taskElement.draggable = true;
+    taskElement.dataset.index = task.index;
+
+    const taskCheckbox = document.createElement('input');
+    taskCheckbox.type = 'checkbox';
```
Comment (Shedrack-Sunday) [inline src/index.js]: - [ ] You seems to have went overboard in your project to implement future functionality. However, for this milestone, you're been tested for certain skills and   its ideal that you at least stick to that.

- [ ]  Based on project requirements, you're to  create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
- description [string].
- completed [bool].
- index: [number].

Right now this is done dynamically, which is good but a future functionality. Kindly follow the instructions to show the required skills been tested here

## Instructions
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @harshi0102        👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlightt
✔️ good PR summary
✔️ Nice UI
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, the changes required below need to be implemented :
- You went overboard to make advance functionality to the project which would be required in future. To test your skills now, you need to apply the static rending of data from a created array as asked on the project requirements instead of dynamic rending based on user input.

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by harshi0102** [comment]
As per last review feedback, based on project requirements, I created an index.js file and set an array of some simple to-do tasks (array of objects). Each task object  contain three keys:

description [string].

completed [bool].

index: [number].

**Review by IngDavidTM — APPROVED** [review]
Hi @harshi0102 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**